pkg/deploy/api/v1/conversion.go

```diff
@@ -288,32 +127,10 @@ func convert_api_RollingDeploymentStrategyParams_To_v1_RollingDeploymentStrategy
 }
 
 func init() {
-	err := api.Scheme.AddDefaultingFuncs(
-		func(obj *DeploymentTriggerImageChangeParams) {
```

Comment:
It's still there, just a different (more appropriate?) place: https://github.com/kargakis/origin/blob/621b77dcffefcaa019f27b11c462d704d03430e7/pkg/deploy/api/v1/defaults.go#L63